nomad/fsm.go

```diff
@@ -429,7 +429,22 @@ func (n *nomadFSM) applyDrainUpdate(reqType structs.MessageType, buf []byte, ind
 		panic(fmt.Errorf("failed to decode request: %v", err))
 	}
 
-	if err := n.state.UpdateNodeDrain(reqType, index, req.NodeID, req.DrainStrategy, req.MarkEligible, req.UpdatedAt, req.NodeEvent); err != nil {
+	accessorId := ""
```

Comment:
One of the fields on `LastDrain` is the accessor ID of the ACL token for the operation. I elected to look it up from the exiting `req.AuthToken` field, instead of plumbing  it through separately.